site/src/request_handlers/github.rs

```diff
@@ -123,8 +123,15 @@ async fn handle_rust_timer(
         let exclude = captures.get(2).map(|v| v.as_str());
         let runs = captures.get(3).and_then(|v| v.as_str().parse::<i32>().ok());
         {
+            let profile;
```

Comment:
I'm not sure about the integration of clippy into the main dashboard. I'd rather leave this part of the PR out for now, and just focus on data gathering.

Reply:
Would the data still be viewable outside the dashboard somewhere?

Reply:
It would be visible only in the local DB. I would like to first merge support for benchmarking clippy. We can discuss its inclusion in the server after that :)